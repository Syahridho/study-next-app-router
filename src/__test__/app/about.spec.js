import { render } from "@testing-library/react";
import AboutPage from "@/app/about/page";
import AboutLayout from "@/app/about/layout";

it("renders homepage unchanged", () => {
  const { container } = render(
    <AboutLayout>
      <AboutPage />
    </AboutLayout>
  );
  expect(container).toMatchSnapshot();
});
