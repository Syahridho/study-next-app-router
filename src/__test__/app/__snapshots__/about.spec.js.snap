// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <nav
    class="fixed left-0 top-0 z-10 h-screen w-60 bg-gray-800"
  >
    <ul
      class="text-white"
    >
      <li>
        Home
      </li>
      <li>
        About
      </li>
      <li>
        Profile
      </li>
    </ul>
  </nav>
  <div>
    <div>
      <h1
        class="text-center"
      >
        About page
      </h1>
    </div>
  </div>
</div>
`;
